projects/ui-particles-angular/index.scss

```diff
@@ -13,4 +13,4 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@forward './src/scss/index.scss';
+@forward './src/scss/index';
```

Comment:
why did you change this ?

Reply:
I think one of lints made me to do that. I can revert